### PR TITLE
Fix object-fit on IE and Edge

### PR DESCRIPTION
**Pull request by jfranciscosousa**

### CODE REVIEW

File: src/components/Background/index.css [hunk 1]
```diff
@@ -25,15 +25,28 @@
...
+
+  @supports not (object-fit: cover) {
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
```
Comment (filipebfernandes) [inline src/components/Background/index.css]: I am confused xD you are using `object-fit: cover` inside  `@supports not (object-fit: cover)`?

Comment (taniaprocha) [inline src/components/Background/index.css]: If you don't support `object-fit: cover` this line is useless right?

Comment (jfranciscosousa) [reply src/components/Background/index.css]: Was a mistake

Comment (jfranciscosousa) [reply src/components/Background/index.css]: Fixed

File: src/components/Background/index.css [hunk 2]
```diff
@@ -25,15 +25,28 @@
...
-  right: 0;
-  bottom: 0;
-  left: 0;
+  top: -50%;
+  left: -50%;
 
-  width: 100%;
-  height: 100%;
+  width: 200%;
+  height: 200%;
```
Comment (taniaprocha) [inline src/components/Background/index.css]: Please rebase with master, I think this is stretch too much the video, but I want to see with the new videos.

Comment (jfranciscosousa) [reply src/components/Background/index.css]: It's a hack for Edge and IE, no other way to do it.

Comment (taniaprocha) [reply src/components/Background/index.css]: Ah ok, only if object-fit not supported

**Pull request merged**